Sources/SpeziSchedulerUI/TodayList.swift

```diff
@@ -0,0 +1,105 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+
+import SpeziScheduler
+import SpeziViews
+import SwiftUI
+
+
+/// An overview of all task for today.
+///
+/// The view renders all task occurring today in a list view.
+///
+/// ```swift
+/// TodayList { event in
+///     InstructionsTile(event) {
+///         QuestionnaireEventDetailView(event)
+///     } action: {
+///         event.complete()
+///     }
+/// }
+///     .navigationTitle("Schedule")
+/// ```
+public struct TodayList<Tile: View>: View {
```

Comment:
Very cool reusable mechanism; we might want to extend this to display any arbitrary date in the future that can be driven by a nice UI adding a date selector 👍

Reply:
Rename the `TodayList` to `EventScheduleList` and allow to specify the date that is shown (with today being the default).